api/scripts/certification/rescore-certifications.js

```diff
@@ -0,0 +1,43 @@
+import 'dotenv/config';
+
+import * as url from 'node:url';
+
+import { disconnect, knex } from '../../db/knex-database-connection.js';
+import { CertificationRescoringByScriptJob } from '../../lib/infrastructure/jobs/certification/CertificationRescoringByScriptJob.js';
+import { logger } from '../../src/shared/infrastructure/utils/logger.js';
+
+const modulePath = url.fileURLToPath(import.meta.url);
+const isLaunchedFromCommandLine = process.argv[1] === modulePath;
+
+async function main(certificationCourseIds) {
+  await _scheduleRescoringJobs(certificationCourseIds);
```

Comment:
Je propose quand même d'informer des erreurs de publication

```suggestion
  const jobs = await _scheduleRescoringJobs(certificationCourseIds);
  
  const errors = jobs
    .filter(result => result.status === 'rejected' );
  
  if(errors.length) {
    errors.forEach((result) => logger.error('Some jobs could not be published', result.reason));
    
    // On informe la IIFE que on est pas bon
    return 1;
  }
  
  // On informe la IIFE qu'on est bon
  return 0;
```